applications/charter/P4_Apps_WG_charter.mdk

```diff
@@ -42,13 +52,14 @@ we propose the WG produce the following deliverables initially.
 - Specifications for common telemetry-report format and semantics
 - Test applications for the above
 
-The specifications will be published at [P4.org](https://p4lang.github.io/p4-spec/)
+The specifications will be published at
+[P4.org](https://p4lang.github.io/p4-spec/)
 and the sample codes and test cases will be available to download at
```

Comment:
"sample codes" is not grammatically correct. How about "source code and test cases will be made available at ..."